learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,65 @@
+
+Select the Vercel account or team and the project you want to add the integration to. You have the flexibility to add it to all your projects or select specific ones (one or multiple projects).
+
+![Form to add Meilisearch Cloud integration, the "Specific projects" option is selected](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/04.add-meilisearch-cloud-form.png)
+
+Click on **Continue**. It will display the list of rights the integration needs to work properly, you can then click on **Add Integration**.
+
+Next, it will take you to the Meilisearch Cloud page. Log in or create an account. New accounts enjoy a 14-day free trial period.
+
+You can choose an existing project or create a new one. To do so, complete the form with the project name and the region.
+
+![Meilisearch Cloud form to create a project complete, with "search-app" as the project's name and "Frankfurt" as the region](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/05.create-a-meilisearch-cloud-project-form.png)
+
+Once you click on **Create project**, you should see the following message:
+“Your Meilisearch + Vercel integration is one click away from being completed.” and a button to finish the setup. On clicking the button, it should redirect you to the integration page in your Vercel Dashboard.
+
+![Meilisearch Cloud integration page in Vercel's dashboard](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/06.meilisearch-cloud-integration-page-in-vercel-dashboard.png)
+
+Meilisearch creates two default API keys: [`Default Search API Key` and `Default Admin API Key`](/learn/security/master_api_keys#using-default-api-keys-for-authorization). With the `Default Admin API Key`, you can control who can access or create new documents, indexes, or change the configuration. The `Default Search API Key` can only be used to access the [search route](/reference/api/search). It is the one you want to use in your front end.
+
+Both keys are automatically added to Vercel along with the Meilisearch host or URL.
+
+You can now go back to your project settings and check the new Meilisearch Environment Variables:
+- `MEILISEARCH_ADMIN_KEY`
+- `MEILISEARCH_URL`
+- `MEILISEARCH_SEARCH_KEY`
+
+![Display the environment variables in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/07.project-settings-environment-variables-tab.png)
+
+The master key–which hasn’t been added to Vercel–grants users full control over an instance, you can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
```

Comment:
This paragraph feels disconnected from the what came before and what comes after. If you want to keep it, give it a bit more context so the user knows what they should do with this key.

If the goal information about an important aspect of Meilisearch security without necessarily integrating it into the regular flow of the guide, consider wrapping it in a `<Capsule></Capsule>` component.

```suggestion
The master key–which hasn’t been added to Vercel–grants users full control over an instance. You can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
```